client/blocks/inline-help/popover.jsx

```diff
@@ -97,6 +113,12 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	switchToClassicEditor = () => {
+		const { siteId, optOut, classicUrl, redirect } = this.props;
+		optOut( siteId );
+		redirect( classicUrl );
```

Comment:
🤔 I see we're already doing this in the other dialog, but it would be more meaningful if we dispatched a single editor preference action, that did analytics calls and navigated as side effects. We'd need to update the navigation middleware to maybe accept information on meta, but that can be left as an optional follow up exercise.